routes/web.php

```diff
@@ -196,6 +196,8 @@
 
         Route::get('/journal/add', 'JournalController@create')->name('create');
         Route::post('/journal/create', 'JournalController@save')->name('save');
+        Route::get('/journal/entries/{entry}/edit', 'JournalController@edit')->name('edit');
+        Route::put('/journal/{entryId}', 'JournalController@update')->name('update');
```

Comment:
It should be
```suggestion
        Route::put('/journal/entries/{entryId}', 'JournalController@update')->name('update');
```
to be more compliant